data/human/human missions.txt

```diff
@@ -7454,3 +7454,213 @@ mission "Adelita 5"
+			choice
+				`	"Uhhh... hand over who?"`
+				`	"I don't negotiate with pirates!"`
+					goto negotiate
+				`	"You can take him, but I don't know where he is."`
+			`	"Bah, you spineless buffoon! Your lies won't save you or that useless salesman. I'll find him in your ship's wreckage!"`
+				goto power
+			label negotiate
+			`	"Aye, then I won't negotiate with you! Prepare to meet your end!"`
+			label power
+			`	The Vanguard's engines roar to life as you see over half a dozen fixed guns take aim at your ship, ready to strike.`
+	npc
+		government "Independent (Killable)"
+		personality heroic staying vindictive
+			confusion 20
+		system "Moktar"
+		ship "Vanguard" "Golden Jewel"
+		on kill
+			set "syndicate bus golden jewel dead"
+			conversation
+				`The crew of the Golden Jewel are no more. You think you can make out the pirate captain drifting in space nearby.`
+				choice
+					`	Best not to dwell on these things.`
+	npc save assist
+		government "Independent"
+		personality derelict target quiet escort
+		system "Moktar"
+		ship "Escape Pod" "Escape Pod 001"
+		on kill
+			conversation
+				`You watch as the escape pod you're racing to save explodes into a cloud of debris, all hope snuffed out in an instant.`
+				choice
+					`	Better not to linger on what could have been.`
+		conversation
+			`You board the escape pod to see its desperate and nearly incoherent occupant: "Oh thank the stars, thank you! Captain, you've got to save me - please! I'm Howard. I promise you a reward if you can fix up this tin can so I can get to <destination>! I'm with Megaparsec, I can make it worth your while!"`
+			choice
+				`	"Sounds good."`
+					decline
+				`	"What's the deal with you and that pirate?"`
+			`	Howard takes a breath and seems to calm himself. "That pirate was a potential investor in the firm I work for. We had a disagreement over the terms and it obviously got quite out of hand. I'll need to find a new business partner now..." He trails off. Time to get back to <planet>.`
+	on complete
+		"reputation: Independent (Killable)" = 10
+		event "end of independent violence"
+		conversation
+			`When you land on <planet>, Howard is waiting for you at the spaceport, looking much more collected than he was on the escape pod. "Thank you, truly. I'm Howard Vendeur, Investor Relations Associate with Megaparsec Incorporated. Anyway, I've managed to scrounge up my savings. It's not much to someone like you, I'm afraid. Just over ninety thousand credits.`
+			`	"However, I see you're clearly a captain of means. I'd have to pull a few strings, but I could get you into a private placement of a Megaparsec Inc perpetual bond. To purchase a perpetual bond of Megaparsec Inc I will need one million credits and for you to sign a few documents. This round is quite generous, with promised returns of 20% per annum expected, of course, to continue into perpetuity, paid out to you as a daily dividend of 500 credits."`
```

Comment:
While this is better grammar, I'm not sure it adds to readability. It's sort of like how "Lo! and behold it." would be a valid sentence, but could confuse people. This could go either way, though.